src/pages/docs/en/basics/data.md

```diff
@@ -28,9 +28,30 @@ Take a look the the [Drive Reference](/docs/en/reference/drive/about) on how to
 
 During development, you can share data between your local environment and your [project](/docs/en/basics/projects) in Space.
 
-To do so, you need to manually connect your development environment to a project's Bases and Drives by using a "project key".
+There is two ways to inject a project key into your development environment:
 
-A project key gives access to Bases and Drives within a project, and can be generated from the "Develop" tab inside your Builder project.
+### Automatic setup
+
+The Space CLI can automatically connect your local development environment with your Space project using the `space dev` command. It will generate a data key on the fly, start your Micro using a command you specify in the `Spacefile` and inject the data key into the process the Micro:
```

Comment:
We should link to the CLI reference here

Reply:
I don't think it is necessary here. Otherwise we should add link to the cli reference in basically every sections of the doc.

Reply:
We do in most cases. If we mention something for the first time on a page we should link to its page if it is important